executor/infoschema_reader_test.go

```diff
@@ -208,7 +209,11 @@ func (s *testInfoschemaTableSuite) TestUserPrivileges(c *C) {
 	c.Assert(len(result.Rows()), Greater, 0)
 }
 
+// mu is used to test DataForTableStatsField and PartitionsTable serially
+var mu sync.Mutex
```

Comment:
It's better to restrict the scope of the variable. I think we can put it in suite.

Reply:
great!